index.html

```diff
@@ -0,0 +1,2199 @@
+              <div class="article-body">
+                <a href="#">
+                  <h2 class="mag_link2 article-title pt-3">
+                    Wuhan Doctor Who First Raised Coronavirus Alarm Defends China's Secrecy
+                  </h2>
+                </a>
+                <p class="card-text">
+                  Zhang Jixian told a state-backed television channel that she 
+                  did not expect the COVID-19 outbreak to be so infectious and
+                   to have mutated into a crippling pandemic.
+                </p>
+              </div>
+            </article>
+            <article class="border-0 featured-articles ">
+              <div>
+                <a href="#">
+                  <img class="rounded-0 w-100 " src="https://d.newsweek.com/en/full/1581097/trump-flag-pollsters-2016-2020.jpg" alt="image">
+                </a>
+                <a href="#">
+                  <p class="politics">POLITICS</p>
+                </a>
+              </div>
+              <div class="article-body">
+                <a href="#">
+                  <h2 class="mag_link2 article-title pt-3">
+                    Wuhan Doctor Who First Raised Coronavirus Alarm Defends China's Secrecy
+                  </h2>
+                </a>
+                <p class="card-text">
+                  Zhang Jixian told a state-backed television channel that she 
+                  did not expect the COVID-19 outbreak to be so infectious and
+                   to have mutated into a crippling pandemic.
+                </p>
+              </div>
+            </article>
+            <article class="border-0 featured-articles ">
+              <div>
+                <a href="#">
+                  <img class="rounded-0 w-100" src="https://d.newsweek.com/en/full/1581097/trump-flag-pollsters-2016-2020.jpg"  alt="image">
+                </a>
+                <a href="#">
+                  <p class="politics">POLITICS</p>
+                </a>
+              </div>
+              <div class="article-body">
+                <a href="#">
+                  <h2 class="mag_link2 article-title pt-3">
+                    Wuhan Doctor Who First Raised Coronavirus Alarm Defends China's Secrecy
+                  </h2>
+                </a>
+                <p class="card-text">
+                  Zhang Jixian told a state-backed television channel that she 
+                  did not expect the COVID-19 outbreak to be so infectious and
+                   to have mutated into a crippling pandemic.
+                </p>
+              </div>
+            </article>
+            <article class="border-0 featured-articles ">
+              <div>
+                <a href="#">
+                  <img class="rounded-0 w-100 " src="https://d.newsweek.com/en/full/1581097/trump-flag-pollsters-2016-2020.jpg"  alt="image">
+                </a>
+                <a href="#">
+                  <p class="politics">POLITICS</p>
+                </a>
```

Comment:
The tag should be positioned on the bottom of the image. Yours have a little space.
![Screenshot from 2020-04-22 12-59-59](https://user-images.githubusercontent.com/51926937/80018440-a0281480-849b-11ea-855d-eeaaaec4afc7.png)